scripts/wood_density.py

```diff
@@ -37,6 +37,11 @@ def __init__(self, **kwargs):
         2. Provide recognition of the efforts of this group in the assembly of the data by using the citation for the database above.
         3. Recognize that these data were assembled by the group for various analyses and research questions. If any of these uses overlap with your interests, you recognize that group has precedence in addressing these questions."""
 
+        if VERSION < 2.1.dev:
```

Comment:
So when we compare versions we can use these packages `from pkg_resources import parse_version` , `from distutils.version import LooseVersion`
I am not sure which is better, but here are some examples  
1

```
from pkg_resources import parse_version
if parse_version(VERSION) < parse_version('2.1.dev'):
```



2
```
import sys
from distutils.version import LooseVersion

a, b = LooseVersion(VERSION), LooseVersion('2.1.dev')

if a > b:
    print a
elif b > a:
    print b
elif a == b:
    print 'equal'


```